docs/user_documentation/admin-features/guide-ref-security.md

```diff
@@ -0,0 +1,102 @@
+# Setup security in MOLGENIS
```

Comment:
Screenshots should only show the important bits, I think. Right now the images in this section contain too much white space for my taste. Also, the menu bar can be removed in most of the images.